tests/core/test_examples.py

```diff
@@ -35,46 +35,94 @@
         stories,
         os.path.join(p, "models", "dialogue"),
         endpoints=endpoints,
-        policy_config="rasa/cli/default_config.yml",
+        policy_config="examples/formbot/config.yml",
     )
-    response = {
-        "events": [
-            {"event": "form", "name": "restaurant_form", "timestamp": None},
-            {
-                "event": "slot",
-                "timestamp": None,
-                "name": "requested_slot",
-                "value": "cuisine",
-            },
-        ],
-        "responses": [{"template": "utter_ask_cuisine"}],
-    }
-
-    with aioresponses() as mocked:
-        mocked.post(
-            "https://example.com/webhooks/actions", payload=response, repeat=True
-        )
-
-        responses = await agent.handle_text("/request_restaurant")
-
-        assert responses[0]["text"] == "what cuisine?"
-
-    response = {
-        "error": "Failed to validate slot cuisine with action restaurant_form",
-        "action_name": "restaurant_form",
-    }
-
-    with aioresponses() as mocked:
-        # noinspection PyTypeChecker
-        mocked.post(
-            "https://example.com/webhooks/actions",
-            repeat=True,
-            exception=ClientResponseError(400, "", json.dumps(response)),
-        )
-
-        responses = await agent.handle_text("/chitchat")
-
-        assert responses[0]["text"] == "chitchat"
+
+    async def mock_form_happy_path(input_text, output_text, slot=None):
```

Comment:
should revert this after merging https://github.com/RasaHQ/rasa/pull/5468/files ?

Reply:
I would keep that, the previous test only tested 1 case, I'd keep it, since these tests are complimentary